ballerina/msg_listener.bal

```diff
@@ -33,7 +33,7 @@ public type StoreListenerConfiguration record {|
     # If true, the message will be acknowledged with failure after the maximum number of retries is reached
     boolean ackWithFailureAfterMaxRetries = false;
```

Comment:
Shouldn't we report a nack (make ackWithFailureAfterMaxRetries = true) by default? Otherwise, the failure to deliver will go unnoticed unless the user explicitly pass ackWithFailureAfterMaxRetries = true.

Reply:
Fixed with [4329897](https://github.com/ballerina-platform/module-ballerina-messaging/pull/3/commits/4329897e0ac9cfe60f6bd76af715eb7c1294d72a)